libs/SalesforceAnalytics/src/com/salesforce/androidsdk/analytics/security/Encryptor.java

```diff
@@ -58,67 +55,8 @@
     private static final String US_ASCII = "US-ASCII";
     private static final String PREFER_CIPHER_TRANSFORMATION = "AES/CBC/PKCS5Padding";
     private static final String MAC_TRANSFORMATION = "HmacSHA256";
-    private static String bestCipherAvailable;
-    private static boolean isFileSystemEncrypted;
-
-    /**
-     * Initializes this module.
-     *
-     * @param ctx Context.
-     * @return True - if the cryptographic module was successfully initialized, False - otherwise.
-     */
-    public static boolean init(Context ctx) {
-
-    	// Checks if file system encryption is available and active.
-        DevicePolicyManager devicePolicyManager = (DevicePolicyManager) ctx.getSystemService(Service.DEVICE_POLICY_SERVICE);
-        isFileSystemEncrypted = devicePolicyManager.getStorageEncryptionStatus() == DevicePolicyManager.ENCRYPTION_STATUS_ACTIVE;
-
-        // Make sure the cryptographic transformations we want to use are available.
-        bestCipherAvailable = null;
-        try {
-            getBestCipher();
-        } catch (GeneralSecurityException gex) {
-            SalesforceAnalyticsLogger.e(ctx, TAG, "Security exception thrown", gex);
-        }
-        if (bestCipherAvailable == null) {
-            return false;
-        }
-        try {
-            Mac.getInstance(MAC_TRANSFORMATION, "BC");
-        } catch (GeneralSecurityException e) {
-            SalesforceAnalyticsLogger.e(ctx, TAG, "No MAC transformation available", e);
-            return false;
-        }
-        return true;
-    }
-
-    private static Cipher getBestCipher() throws GeneralSecurityException {
```

Comment:
Same reasons as listed above. The best cipher is the one provided by the OS now, i.e. `Conscrypt` on `Android P` and higher.